content/en/user-manual/graphics/shader-chunk-migrations.md

```diff
@@ -29,9 +29,101 @@ By doing this you will no longer see warning messages in the console.
+    vec3 specularity;
+};
+```
+
+| Chunk | Changes |
+| --- | --- |
+| `ambient(Constant/Env/SH)` | <ul><li>Accepts a vec3 for the world normal instead of using `dNormalW`</li></ul>
+| `aoDiffuseOcc` | <ul><li>Accepts a float value for the AO, instead of using `dAO`</li></ul>
+| `aoSpec(Occ/OccConst/OccConstSimple/OccSimple)` | <ul><li>Accepts float gloss, float ao and a vec3 world normal instead of using `dGlossiness`, `dAo` and `dNormalW` respectively</li></ul>
+| `combine` | <ul><li>Accepts vec3 for albedo, sheen specularity and a float for clearcoat specularity instead of using `dAlbedo`, `sSpecularity` and `ccSpecularity`</li></ul>
+| `clusteredLight` | <ul><li>Reliance on globals have been reduced to only use dReflDirW and ccReflDirW</li></ul>
+| `end` | <ul><li>Passes albedo, sheen specularity and clearcoat specularity to combine using `litShaderArgs`, uses `litShaderArgs.emission` instead of relying on `dEmission`</li></ul>
+| `fresnelSchlick` | <ul><li>Accepts gloss and `IridescenceArgs` instead of relying on `dGlossiness`, `dIridescenceFresnel` and `dIridescence`</li></ul>
+| `iridescenceDiffraction` | <ul><li>Accepts a float as iridescence thickness instead of using `dIridescenceThickness`</li></ul>
+| `lightDiffuseLambert` | <ul><li>Accepts vec3 world normal instead of using `dNormalW`</li></ul>
+| `lightSheen` | <ul><li>Accepts vec3 world normal, float gloss instead of relying on `dNormalW`, `dGlossiness`</li></ul>
+| `lightSpecular(AnisoGGX/Blinn/Phong)` | <ul><li>Accepts vec3 reflection direction, vec3 world normal and float gloss instead of relying on `dGlossiness`, `dNormalW` and `dReflDirW`</li></ul>
+| `lightmap(DirAdd/Add)` | <ul><li>Accepts a vec3 lightmap direction, vec3 lightmap value, vec3 world normal, float gloss, vec3 specularity and IridescenceArgs instead of relying on `dLightMap`, `dLightmapDir`, `dNormalW` and `dSpecularity`</li></ul>
+| `metalnessModulate` | <ul><li>Accepts a `LitShaderArguments` struct which is updated by the chunk. Removes the reliance on `dSpecularity`, `dMetalness` and `dAlbedo`</li></ul>
+| `output(Alpha/AlphaPremul)` | <ul><li>Uses `litShaderArgs.opacity` instead of `dAlpha`</li></ul>
+| `reflDir(Aniso)` | <ul><li>Accepts a vec3 world normal and a float value for gloss, instead of using `dGlossiness` and `dNormalW`</li></ul>
+| `reflection(CC/Cube/Env/EnvHQ/Sphere/SphereLow)` | <ul><li>Accepts a vec3 reflection direction and a float gloss value instead of using `dReflDirW`/`ccReflDirW` and `dGlossiness`</li></ul>
+| `reflectionSheen` | <ul><li>Accepts a vec3 world normal and a float gloss value instead of using `dNormalW` and `sGlossiness`</li></ul>
+| `refraction(Cube/Dynamic)` | <ul><li>Accepts a vec3 world normal, float thickness, float gloss, vec3 specularity, vec3 albedo, float transmission and `IridescenceArgs` instead of using `dNormalW`, `dAlbedo`, `dTransmission`, `dThickness`, `dGlossiness`, `dSpecularity` and passes the iridescence arguments to the fresnel function</li></ul>
```

Comment:
Looks like each table row is missing the trailing `|`

Reply:
Could you please build the manual locally, and attach the screenshot of the new section in the PR description.

Reply:
Looks like the trailing `|` is still missing.